packages/web-components/fast-foundation/src/number-field/number-field.ts

```diff
@@ -267,8 +275,13 @@ export class FASTNumberField extends FormAssociatedNumberField {
      */
     public stepUp(): void {
         const value = parseFloat(this.value);
+        const decimalString = this.step.toString();
+        const decimalPlaces = decimalString.includes(".")
+            ? decimalString.split(".")[1].length
```

Comment:
This hard codes `.` as the decimal point, but what about i18n? Sometimes decimal point is `,` for instance...